<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>900</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>47</green>
        <blue>62</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>47</green>
        <blue>62</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>47</green>
        <blue>62</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>47</green>
        <blue>62</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>47</green>
        <blue>62</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>47</green>
        <blue>62</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>47</green>
        <blue>62</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>47</green>
        <blue>62</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>47</green>
        <blue>62</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MAZE by gconn, torell &amp; cflossie</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(32, 47, 62);
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>961</width>
      <height>871</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
	border: 1px solid #4fbcff;
	background-color: #687884;
	color: #FFFAFF;
	border-radius: 25px;
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_maze">
     <attribute name="title">
      <string>MAZE</string>
     </attribute>
     <widget class="WidgetMaze" name="widgetMaze" native="true">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>500</width>
        <height>500</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #4fbcff;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadMaze">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>610</y>
        <width>431</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 1px solid #4fbcff;
	background-color: rgb(19, 37, 89);
	color: rgb(255, 250, 255);
	border-radius: 25px;
}

QPushButton:pressed {
     background-color: rgb(79, 188, 255);
 }

QPushButton:hover {
     background-color: #4fbcff;
 }</string>
      </property>
      <property name="text">
       <string>LOAD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generateMaze">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>700</y>
        <width>411</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 1px solid #4fbcff;
	background-color: rgb(19, 37, 89);
	color: rgb(255, 250, 255);
	border-radius: 25px;
}

QPushButton:pressed {
     background-color: rgb(79, 188, 255);
 }

QPushButton:hover {
     background-color: #4fbcff;
 }</string>
      </property>
      <property name="text">
       <string>GENERATE</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="heightMaze">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>635</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
	border: 1px solid #4fbcff;
	background-color: #687884;
	color: #FFFAFF;
	border-radius: 25px;
}</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="widthMaze">
      <property name="geometry">
       <rect>
        <x>699</x>
        <y>635</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
	border: 1px solid #4fbcff;
	background-color: #687884;
	color: #FFFAFF;;
	border-radius: 25px;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>610</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #FFFAFF;
}</string>
      </property>
      <property name="text">
       <string>HEIGHT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="width1">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>610</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #FFFAFF;
}</string>
      </property>
      <property name="text">
       <string>WIDTH</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_cave">
     <attribute name="title">
      <string>CAVE</string>
     </attribute>
     <widget class="WidgetCave" name="widgetCave" native="true">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>500</width>
        <height>500</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #4fbcff;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generateCave">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>610</y>
        <width>241</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 1px solid #4fbcff;
	background-color: rgb(19, 37, 89);
	color: rgb(255, 250, 255);
	border-radius: 25px;
}

QPushButton:pressed {
     background-color: rgb(79, 188, 255);
 }

QPushButton:hover {
     background-color: #4fbcff;
 }</string>
      </property>
      <property name="text">
       <string>GENERATE</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="heightCave">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>610</y>
        <width>201</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
	border: 1px solid #4fbcff;
	background-color: #687884;
	color: #FFFAFF;;
	border-radius: 25px;
}</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="widthCave">
      <property name="geometry">
       <rect>
        <x>699</x>
        <y>610</y>
        <width>201</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
	border: 1px solid #4fbcff;
	background-color: #687884;
	color: #FFFAFF;;
	border-radius: 25px;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="width2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>588</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #FFFAFF;
}</string>
      </property>
      <property name="text">
       <string>WIDTH</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="loadCave">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>690</y>
        <width>241</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 1px solid #4fbcff;
	background-color: rgb(19, 37, 89);
	color: rgb(255, 250, 255);
	border-radius: 25px;
}

QPushButton:pressed {
     background-color: rgb(79, 188, 255);
 }

QPushButton:hover {
     background-color: #4fbcff;
 }</string>
      </property>
      <property name="text">
       <string>LOAD</string>
      </property>
     </widget>
     <widget class="QLabel" name="hightLabel">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>588</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #FFFAFF;
}</string>
      </property>
      <property name="text">
       <string>HEIGHT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="birthLabel">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>755</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #FFFAFF;
}</string>
      </property>
      <property name="text">
       <string>BIRTH LIMIT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="deathLabel">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>755</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #FFFAFF;
}</string>
      </property>
      <property name="text">
       <string>DEATH LIMIT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="death">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>690</y>
        <width>201</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
	border: 1px solid #4fbcff;
	background-color: #687884;
	color: #FFFAFF;;
	border-radius: 25px;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="birth">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>690</y>
        <width>201</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
	border: 1px solid #4fbcff;
	background-color: #687884;
	color: #FFFAFF;;
	border-radius: 25px;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
     <widget class="QRadioButton" name="autoMode">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
	color: #FFFAFF;
}</string>
      </property>
      <property name="text">
       <string>AUTO MODE</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="stepMode">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>625</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
	color: #FFFAFF;
}</string>
      </property>
      <property name="text">
       <string>STEP MODE</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="nextStep">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>690</y>
        <width>201</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 1px solid #4fbcff;
	background-color: rgb(19, 37, 89);
	color: rgb(255, 250, 255);
	border-radius: 25px;
}

QPushButton:pressed {
     background-color: rgb(79, 188, 255);
 }

QPushButton:hover {
     background-color: #4fbcff;
 }</string>
      </property>
      <property name="text">
       <string>NEXT STEP</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="timer">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>690</y>
        <width>201</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
	border: 1px solid #4fbcff;
	background-color: #687884;
	color: #FFFAFF;;
	border-radius: 25px;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100000000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_wolf">
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <attribute name="title">
      <string>TEAM</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>60</y>
        <width>451</width>
        <height>591</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border: 1px solid #4fbcff;
};</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../view/resurses.qrc">:/img/wolf_pack2.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>670</y>
        <width>751</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #4fbcff;
}</string>
      </property>
      <property name="text">
       <string>&quot;Дикие леса Канады полны лютоволков! Ты знаешь о лютоволках, не так ли?...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>681</y>
        <width>511</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>33</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #4fbcff;
}</string>
      </property>
      <property name="text">
       <string>Они как волки, только лютые!&quot;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>921</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>33</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #4fbcff;
}</string>
    </property>
    <property name="text">
     <string>SO AMAZE MAZE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetCave</class>
   <extends>QWidget</extends>
   <header>View/widgetCave.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetMaze</class>
   <extends>QWidget</extends>
   <header location="global">View/widgetMaze.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../view/resurses.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
